### ISSUE TITLE
citation details absent

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#1101]
https://www.pombase.org/reference/PMID:37453417

**Comment by kimrutherford** [comment]
Hi Val.  That was a weird one.  I think I've tracked down the problem.

It seems the issue is that there is a publication in Chado with a semicolon at the end of the ID: "PMID:18641648;".  It's also there in Chado without the semicolon so the publication page exists: https://www.pombase.org/reference/PMID:18641648

The semicolon is causing the request to PubMed to fail so the publication details for new publications aren't being fetched.  There are 8 publications that have no details because of this:

```
PMID:37425826
PMID:37431652
PMID:37459529
PMID:37453417
PMID:37445861
PMID:37459529
PMID:37463585
PMID:37446379
```

I've fixed the code that fetches the details from PubMed to remove the semicolon so it should be OK on Sunday morning.

The ID with the semicolon is from chromosome1.contig:
```
FT   snoRNA          complement(1966483..1966635)
FT                   /db_xref="PMID:18641648;"
```

I'm about to fix the loading code to clean up or complain about PMIDs that don't look correct.  I'll fix the ID in the contig file on Sunday once I've confirmed that my fix for the loader works.
